modules/ROOT/partials/configuration/mergetags_prefix.adoc

```diff
@@ -0,0 +1,22 @@
+[[mergetags_prefix]]
+== `+mergetags_prefix+`
+
+This option specifies the prefix to be attached to each merge tag.
+
+Whatever this option is set to, when it is typed, it triggers the {pluginname} autocompleter within the editor.
```

Comment:
Or finds things in the HTML being loaded or inserted that matches that prefix together with the suffix.

Reply:
Added a usage note re TinyMCE auto-recognising strings that match an existing <prefix><merge-tag><suffix> when such a string is added to a document.